common/common-rest/src/main/java/io/servicecomb/common/rest/codec/param/QueryProcessorCreator.java

```diff
@@ -26,10 +29,36 @@
 
 import io.servicecomb.common.rest.codec.RestClientRequest;
 import io.swagger.models.parameters.Parameter;
+import io.swagger.models.parameters.QueryParameter;
 
 public class QueryProcessorCreator implements ParamValueProcessorCreator {
+
   public static final String PARAMTYPE = "query";
 
+  public QueryProcessorCreator() {
+    ParamValueProcessorCreatorManager.INSTANCE.register(PARAMTYPE, this);
+  }
+
+  @Override
+  public ParamValueProcessor create(Parameter parameter, Type genericParamType) {
+    JavaType targetType = TypeFactory.defaultInstance().constructType(genericParamType);
+
+    String collectionFormat = ((QueryParameter) parameter).getCollectionFormat();
+
+    if (collectionFormat == null) {
+      return new QueryProcessor(parameter.getName(), targetType);
+    }
+    if (collectionFormat.equals("csv")) {
+      return new CsvQueryProcessor(parameter.getName(), targetType);
+    }
+    if (collectionFormat.equals("multi")) {
+      return new MultiQueryProcessor(parameter.getName(), targetType);
```

Comment:
what's the relationship between MultiQueryProcessor and QueryProcessor
their code are the same.